### PR TITLE
Update zh.json

### PR DESCRIPTION
**Pull request by bAdOoWindy**
from en.json sep 21 2023

### CODE REVIEW

**Comment by bAdOoWindy** [comment]
close old one workflow  in same day

**Comment by dcbroad3** [comment]
Thanks for working on this! Do you intend to translate the entries you've added that are still in English as well?

**Comment by bAdOoWindy** [comment]
sure still in English

**Comment by bAdOoWindy** [comment]
is there any not right?
why take so long

**Comment by dcbroad3** [comment]
> is there any not right? why take so long

Sorry, let me rephrase my earlier question. You've added several entries which are not translated into Chinese - they are still in English. Do you plan to translate those as well? Is there a reason you've left those in English? Either way this PR is an improvement and I'll merge it, I just wanted to make sure you're finished.
